src/adapters/refract/transformResource.coffee

```diff
@@ -105,7 +104,8 @@ module.exports = (resourceElement, location, options) ->
 
       # In refract just here we have method and href
       resource.method = _.chain(httpRequest).get('attributes.method', '').contentOrValue().value()
-      resource.actionUriTemplate = _.chain(httpRequest).get('attributes.href', '').contentOrValue().value()
+      requestUriTemplate = _.chain(httpRequest).get('attributes.href', '').contentOrValue().value()
+      resource.actionUriTemplate = requestUriTemplate or transitionUriTemplate
```

Comment:
I am sorry but I don't understand this part. An action can contain many requests. Why are we taking one request here?